spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/controller/JournalControllerTest.java

```diff
@@ -48,4 +50,15 @@ public void test_getJournal() {
 		ClientApplicationEvent event = history.iterator().next();
 		assertThat(event, sameInstance(emittedEvent));
 	}
+
+	@Test
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/joshiste/spring-boot-admin/file/2229709764/issues/source?bid=3175464&fileBranchId=3175465#l55)